### ISSUE TITLE
Add ability to make response parameters flexible 

### ISSUE THREAD

**Issue by obiefy** [obiefy/api-response#7]
I thinking about something like this.
```php
api()->response($status, $message, $data, $extraData);
```
at the current version this will not work, because `response()` method parameters are fixed to three (status, message and data).

**Comment by obiefy** [comment]
Hi, @SamyOteroGlez,
I think in case developer need extra data than (status, message, and data).
for example, if I want my response to be like this:
```JSON
{
    "STATUS": 403,
    "MESSAGE": "Sorry, you cannot access this content",
    "DATA": [
        {"name": "user name"}
    ],
    "code": 30566,
    "reference": "ERROR-2019-09-14"
}
```
In this case the package usage will like this:
```php
api()->reposne(403,
     'Sorry, you cannot access this content', 
     ['name' => 'user name'], 
     'code' => 30566,
     'reference': 'ERROR-2019-09-14'
     );
```
we can use [function variable-arg list](https://www.php.net/manual/en/functions.arguments.php#functions.variable-arg-list.new) for this issue.

**Comment by SamyOteroGlez** [comment]
Sounds good, I was thinking on using the same variable-length argument lists, but I wanted to be sure what you were thinking about it.

Did you already start working on that? I can do it.

Also, have you considered adding a kanban board for this repo? GitHub gives that functionality, so ideas like this one can be added to the board once tasks are defined.

Cheers

**Comment by obiefy** [comment]
To be honest I didn't start working on this issue. I'll be happy if you create PR to make this done.
We can also create GitHub Projects.. to make progress under control.

**Comment by SamyOteroGlez** [comment]
That's awesome!
Please go ahead and create it. 
I'll be working on the new pr for `adding the ability to make response parameters flexible`.

**Comment by obiefy** [comment]
Fixed #14.
